core/storage/question/gae_models.py

```diff
@@ -151,6 +151,8 @@ class QuestionSkillLinkModel(base_models.BaseModel):
     question_id = ndb.StringProperty(required=True, indexed=True)
     # The ID of the skill to which the question is linked.
     skill_id = ndb.StringProperty(required=True, indexed=True)
+    # The description of the skill for frontend display purposes.
+    skill_description = ndb.StringProperty(required=True)
```

Comment:
I think maintaining this property separately might not be such a good idea, we'll always need to keep it in sync with the skill model (e.g. what if the skill's description is updated)?

Can we instead do something where we fetch these models and then do a second get_mutli() to fetch the skill descriptions for the skill IDs in question if needed?

Reply:
Ok, will do that.

Reply:
Done